sdk/core/azure-core/src/main/java/com/azure/android/core/internal/util/serializer/JacksonAdapter.java

```diff
@@ -309,4 +302,22 @@ private String serializeRaw(Object object) {
             return null;
         }
     }
+
+    private static String join(@NonNull CharSequence delimiter, @NonNull Iterable tokens) {
```

Comment:
`TextUtils` is an Android framework class and thus we cannot access its full functionality when running unit tests. Since the alternative `String.join()` is only available for a late Android version, I figured writing a simple method to join would be the best way to get around this problem.